packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -416,7 +426,7 @@ export class OdspSummaryUploadManager {
                         reusedBlobs += result.reusedBlobs;
                         blobs += result.blobs;
                     } else {
-                        if (allowHandleExpansion) {
+                        if (allowHandleExpansion && this.hostPolicy.blobDeduping) {
```

Comment:
It's a bit verbose with this.hostPolicy.blobDeduping, considert passing boolean to this function / class or creating variable in function with it.

Reply:
Added var.